src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -90,11 +112,8 @@ trait BackendCall {
     call.instantiateCommandLine(backendInputs, engineFunctions)
   }
 
-  /**
-   * Block while executing this call.  Return a Success(Map[CallOutputs]) if the
-   * Call ran successfully.  Otherwise return a Failure.
-   */
-  def execute: ExecutionResult
+  /** Initiate execution, callers can invoke `poll` once this `Future` completes successfully. */
+  def execute(implicit ec: ExecutionContext): Future[ExecutionHandle]
```

Comment:
I must have fixed this after you made your comment but before I saw it, looks okay now.  :smile: